### DISCUSSION TITLE
Newline characters or carriage returns

### DISCUSSION THREAD

**Discussion by rynsp8** [UglyToad/PdfPig#661] [Q&A]
When using the Letters method? or class? does it represent Newline characters or carriage returns?  I'm wanting to parse out lines with specific symbols, but I can't seem to find any newline characters.  Maybe there is an end-of-line character I can test against?

**Comment by InusualZ** [comment]
New lines are not included in a PDF.

What you could do is check for the text coordinates.

**Reply by rynsp8** [reply]
Ok, so if there are no Newline characters in a PDF then I'll have to figure it out another way.  

I suppose I can print block to a text file and use regular expressions to pull out whatever I need.  

But, I'm curious, how would text coordinates help?

**Reply by InusualZ** [reply]
If you know that the variable text that you are looking is going to always be at a specific location in a page. You can filter out all text that doesn't intercept with with that box.
